composer.json

```diff
@@ -8,12 +8,13 @@
             "email": "stephen@stecman.co.nz"
         }
     ],
+    "minimum-stability": "beta",
```

Comment:
This won't have any effect, because it's read only from master `composer.json` of your project and not the dependency projects.

Reply:
Removed